src/nrepl/index.ts

```diff
@@ -761,11 +760,11 @@ export class NReplEvaluation {
 
     private _pprintOut: string;
 
-    private _outPut: String;
+    private _outPut: string;
 
-    private _errorOutput: String;
+    private _errorOutput: string;
 
-    private _exception: String;
+    private _exception: string;
```

Comment:
`string` is the correct type here, whereas `String` is a constructor function for `string` types.